yarn.lock

```diff
@@ -4428,13 +4428,13 @@ cozy-authentication@1.19.1:
     snarkdown "1.2.2"
     url-polyfill "1.1.7"
 
-cozy-client@8.1.0:
-  version "8.1.0"
-  resolved "https://registry.yarnpkg.com/cozy-client/-/cozy-client-8.1.0.tgz#3dbaa6e6454241f186e8b242990129f4a22d3334"
-  integrity sha512-Xy4cpxTIdu1jVUetgd6aG+n4qgYKqhXTkxnSq/OPt+Px8kS1XH3aQMqPT/bBb81sM5RGz1qtkPP8H9SHBbNo6g==
+cozy-client@^8.3.1:
```

Comment:
Is it expected not to have pinned the version ?

Reply:
Hmm nop. So using package@latest do no pin the version. Interesting.